### PR TITLE
2.0

### PR DESCRIPTION
**Pull request by sac-urs**

### CODE REVIEW

File: gdeploylib/global_vars.py [hunk 1]
```diff
@@ -31,7 +31,15 @@ class Global(object):
     var_file = None
     master = None
     brick_hosts = []
-    log_file = os.path.expanduser('~/.gdeploy/logs/gdeploy.log')
```
Comment (nandajavarma) [inline gdeploylib/global_vars.py:34]: I am trying to keep away as much as logic as possible from global_vars. global_vars can be used as an easy lookup for all the global variables. That is the idea. Having this huge logic in global bars kind of beat the purpose. Also, logging is altogether an independent framework as per the current architecture. This would mess it up.

File: gdeploylib/__init__.py [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # -*- coding: utf-8 -*-
 
+from global_vars import Global
```
Comment (nandajavarma) [inline gdeploylib/__init__.py:3]: Is this change intentional? I don't understand the purpose.

Comment (sac-urs) [inline gdeploylib/__init__.py:3]: Ignore this. I was observing how `import *' behaves on the ordering of the import statements.

File: gdeploylib/global_vars.py (hunk 1 above)
Comment (sac-urs) [inline gdeploylib/global_vars.py:34]: As of now gdeploy does not work unless ~/.gdeploy/logs is created manually, it spits out Traceback. Move this out of Global, put it anywhere before logging calls open() on the log_file.
